examples/rds.txt

```diff
@@ -0,0 +1,111 @@
+RDS MYSQL AUDIT LOGS:
+    {
+        "messageType": "DATA_MESSAGE",
+        "owner": "908047316593",
+        "logGroup": "/aws/rds/instance/mysql-db-for-logs/audit",
+        "logStream": "mysql-db-for-logs",
+        "subscriptionFilters": [
+            "mysql-audit-filter"
+        ],
+        "logEvents": [
+            {
+                "id": "36283723329239606779976941014300561991466727829296513024",
+                "timestamp": 1627018425000,
+                "message": "20210723 05:33:45,ip-10-1-3-230,rdsadmin,localhost,203,0,CONNECT,,,0,SOCKET"
+            },
+            {
+                "id": "36282733332258008410023818770931443951379879892065320960",
+                "timestamp": 1626974032000,
+                "message": "20210726 12:05:10,ip-10-1-3-230,db_user,157.25.19.100,1029,242734,QUERY,`testdb`,'select * from mysql.user\nLIMIT 0, 1000',1142,,"
+            },
+            {
+                "id": "36282735473129547468963640358360398125622584466185256962",
+                "timestamp": 1626974128000,
+                "message": "20210722 17:15:28,ip-10-1-3-230,admin,157.25.19.100,13,0,DISCONNECT,testdb,,0,SSL"
```

Comment:
AWS can't even be consistent when it comes to having dbname in backticks or not. Love this cloud